README.md

```diff
@@ -29,18 +35,21 @@ addon | version | summary
 
 [//]: # (end addons)
 
-<!-- prettier-ignore-end -->
+Translation Status
+------------------
+[![Translation status](https://translation.odoo-community.org/widgets/web-12-0/-/multi-auto.svg)](https://translation.odoo-community.org/engage/web-12-0/?utm_source=widget)
```

Comment:
Note that you already have a badge for this in the new template:

`[![Translation Status](https://translation.odoo-community.org/widgets/web-14-0/-/svg-badge.svg)](https://translation.odoo-community.org/engage/web-14-0/?utm_source=widget)`